CMakeModules/FindDialyzer.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# This file is part of AtomVM.
+#
+# Copyright 2019 Riccardo Binetti <rbino@gmx.com>
```

Comment:
Is it the proper copyright line?

Reply:
It is a copy of `CMakeModules/FindElixir.cmake` + find and replace. No creative work on top of it.